clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus_test.go

```diff
@@ -49,7 +52,6 @@ var (
 
 func turnOffLoggingToStderrInKlog(t *testing.T) {
 	if klogLogToStderr {
-		klog.InitFlags(nil)
```

Comment:
For some reason having `InitFlags` in `exec_service.go` changed code initialisation in a way that caused `flags.go#init()` (which already has `klog.InitFlags(nil)`) to fire before this line, causing a flag init collision (nasty bug). Solved by adding an empty import to `flags` package.